lib/challenge.js

```diff
@@ -19,6 +19,31 @@
   // - park
 
 // Answer Starts Here
+'Calendar'
+Description: our app uses a calendar to keep track of appointments.
+
+Every 'calendar' can
+- set busy/free status
+- invite another person to an appointment
+
+
+Every 'calendar app' has
+- blocks of time
+- appointments
+- links to external calendars
+
+Each 'block of time' has
+- a minute
+- an hour
+- a day
+- a month
+- a year
+
+Each 'appointment' has
+- has a 'block of time assignment'
+- a 'name'
+- a 'location'
+
```

Comment:
This is plain English in a code file. It should all be commented. Because these are not comments, you can't use the tests that would help you catch minor errors before submitting the assessment -- the script will blow up at the very top from syntax errors.